fuzzing/engines/jazzer_launcher.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The command line arguments for launching fuzz tests run with the libFuzzer
```

Comment:
I see the comment mentions libFuzzer. In a previous revision, you mentioned the launcher interface is the same. Should we reuse the libFuzzer one instead of duplicating it here?

Reply:
Jazzer is compatible with libFuzzer in the sense that it understands all libFuzzer arguments. It does take additional (double-dash) arguments though and I initially forgot that we need to pass it `--reproducer_path` so that Java reproducers are also stored in the artifact dir.

I restructured this file to make this clearer.